app/(afterLogin)/admin/dashboard/page.tsx

```
@@ -232,10 +232,7 @@ const AdminDashboard = () => {
   const activeSubscriptionData = subscriptionsData?.items?.find(
     (sub) => sub.isActive === true,
   );
-  console.log(
-    { plansWithSameCurrency, activeSubscriptionData },
-    'plansWithSameCurrency',
-  );
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)